src/covid.c

```diff
@@ -360,7 +362,9 @@ int init_covid()
 	check_keys(NULL);
 
 	int err = 0;
+	#ifndef NATIVE_POSIX
```

Comment:
Please introduce another configuration to enable/disable BT functionality as for some the native posix build could actually use the host's bluetooth stack.

Reply:
@hendrick404 This issue requires another configuration variable, you could just use CONFIG_BT of Zephyr?